VTEX - Catalog API.json

```diff
@@ -14460,8 +14656,8 @@
                 "tags": [
                     "Subcollection"
                 ],
-                "summary": "Create SubCollection",
-                "description": "Creates a new SubCollection inclusion or exclusion under a Collection.",
+                "summary": "Create Subcollection",
+                "description": " >⚠️ There are two ways to configure collections, through Legacy CMS Portal or using the Beta Collection module. This endpoint is compatible with [collections configured through the Legacy CMS Portal](https://help.vtex.com/en/tutorial/adding-collections-cms--2YBy6P6X0NFRpkD2ZBxF6L).\n\nCreates a new Subcollection, which is a [Group](https://help.vtex.com/tutorial/creating-a-product-collection--tutorials_244#creation-of-groups-by-department-category-or-sub-category) within a  Collection. A Subcollection can be either “Exclusive” (all the products contained in it will not be used) or “Inclusive” (all the products contained in it will be used).\r\n## Request body example\r\n\r\n```json\r\n{\r\n    \"CollectionId\": 149,\r\n    \"Name\": \"Test\",\r\n    \"Type\": \"Exclusive\",\r\n    \"PreSale\": true,\r\n    \"Release\": false\r\n}\r\n```\r\n## Response body example\r\n\r\n```json\r\n{\r\n    \"Id\": 13,\r\n    \"CollectionId\": 149,\r\n    \"Name\": \"Test\",\r\n    \"Type\": \"Exclusive\",\r\n    \"PreSale\": true,\r\n    \"Release\": false\r\n}\r\n```",
```

Comment:
```suggestion
                "description": " >⚠️ There are two ways to configure collections, through Legacy CMS Portal or using the Beta Collection module. This endpoint is compatible with [collections configured through the Legacy CMS Portal](https://help.vtex.com/en/tutorial/adding-collections-cms--2YBy6P6X0NFRpkD2ZBxF6L).\n\nCreates a new Subcollection, which is a [Group](https://help.vtex.com/en/tutorial/creating-a-product-collection--tutorials_244#creation-of-groups-by-department-category-or-sub-category) within a  Collection. A Subcollection can be either “Exclusive” (all the products contained in it will not be used) or “Inclusive” (all the products contained in it will be used).\r\n## Request body example\r\n\r\n```json\r\n{\r\n    \"CollectionId\": 149,\r\n    \"Name\": \"Test\",\r\n    \"Type\": \"Exclusive\",\r\n    \"PreSale\": true,\r\n    \"Release\": false\r\n}\r\n```\r\n## Response body example\r\n\r\n```json\r\n{\r\n    \"Id\": 13,\r\n    \"CollectionId\": 149,\r\n    \"Name\": \"Test\",\r\n    \"Type\": \"Exclusive\",\r\n    \"PreSale\": true,\r\n    \"Release\": false\r\n}\r\n```",
```